unit-tests/live/intrinsics/test-motion-intrinsics.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*)
+
+import pyrealsense2 as rs
+from rspy import test
+
+
+device = test.find_first_device_or_exit()
+motion_sensor = device.first_motion_sensor()
+
+if motion_sensor:
+
+    motion_profile_accel = next(p for p in motion_sensor.profiles if p.stream_type() == rs.stream.accel)
+    if motion_profile_accel:
+        motion_profile_accel = motion_profile_accel.as_motion_stream_profile()
+        intrinsics_accel = motion_profile_accel.get_motion_intrinsics()
+
+        # Test #1
+        test.start('Check intrinsics in motion sensor - ACCEL profile')
+        test.check(len(str(intrinsics_accel)) > 0)
```

Comment:
What is this check?
You convert to string and check the length?
Why?

Reply:
Also add the above code into the test, it's part of the test, we want to see this flow is working.
and for example check
test.check( motion_profile_accel )

Reply:
> What is this check? You convert to string and check the length? Why?

The intrinsic has `data: [[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0]], noise_variances: [0, 0, 0], bias_variances: [0, 0, 0]` data.
So we sought if it not empty it enough goof for us. May we improve it?

Reply:
When will you get an empty string?
IMO never.
Even if all 0.
Am I wrong?

Reply:
But you know what, its interesting, can you add a log.d of the string? so we can see how it looks like?
And add comment about why you check it as string

Reply:
> When will you get an empty string? IMO never. Even if all 0. Am I wrong?

I think you are right, if no intrinsic it failing before.